build.gradle

```diff
@@ -16,4 +17,22 @@ test {
         events "failed"
         exceptionFormat "full"
     }
-}
```

Comment:
A blank line is missing.

Reply:
Talked with Joel, there is no blank line missing. No need for changes